### ISSUE TITLE
The requested URL /oauth/authorize was not found on this server.

### ISSUE THREAD

**Issue by lesandie** [klassdata/moodle-local_smart_klass#4]
Hi,
Installed on a moodle 2.7.1 following the guide and we're getting this error when we click on the Configure access to SmartKlass dashboard. Also, change the AllowOverride directive to All but with no success and checked permissions. Something more that needs to be installed?
Thanks in advance

**Comment by michaelrabelo** [comment]
Same problem here! Any clues of what is happening?

**Comment by michaelrabelo** [comment]
My Moodle version is 3.0

**Comment by lesandie** [comment]
Just edit libraries/h5p.classes.php and comment both methods
getLibraryStats abd getNumAuthors

that should do it

On Wed, Apr 20, 2016 at 7:24 PM, michael.douglas@outlook.com <
notifications@github.com> wrote:

> My Moodle version is 3.0
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/klassdata/moodle-local_smart_klass/issues/4#issuecomment-212523298

## 

Lt. Col. Sandie

**Comment by lesandie** [comment]
Sorry wrong group :-(

On Fri, Apr 22, 2016 at 6:01 PM, Le Sandie lesandie@gmail.com wrote:

> Just edit libraries/h5p.classes.php and comment both methods
> getLibraryStats abd getNumAuthors
> 
> that should do it
> 
> On Wed, Apr 20, 2016 at 7:24 PM, michael.douglas@outlook.com <
> notifications@github.com> wrote:
> 
> > My Moodle version is 3.0
> > 
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly or view it on GitHub
> > https://github.com/klassdata/moodle-local_smart_klass/issues/4#issuecomment-212523298
> 
> ## 
> 
> Lt. Col. Sandie

## 

Lt. Col. Sandie

**Comment by thalesmsousa** [comment]
I have the same problem. Were you able to resolve?

**Comment by thalesmsousa** [comment]
I have the same problem. Were you able to resolve?

**Comment by thalesmsousa** [comment]
I have the same problem. Were you able to resolve?

**Comment by thalesmsousa** [comment]
I have the same problem. Were you able to resolve?

**Comment by dhyre-mdl** [comment]
I'm having the exact same problem. I'll try the edit above.

**Comment by dhyre-mdl** [comment]
Sorry, that doesn't apply to my problem Has anyone solved this one yet?
![screenhunter_685 may 10 09 16](https://cloud.githubusercontent.com/assets/19275694/15132004/02c46608-1690-11e6-834a-f04f3dcf9b08.jpg)

**Comment by RicardoGuerra** [comment]
Hi, I have the same problem... someone has a solution?

**Comment by dhyre-mdl** [comment]
Gosh, I hope so. If Klass can't buy more server power, I hope they tell us
soon.

**Comment by dhyre-mdl** [comment]
Yep me too and debugging is giving me another error telling me I need to change a SmartKlass related name somewhere. Are the Smartklass people still in business or have they given up?

**Comment by alaaelgndy** [comment]
please ...

i have a problem with smartklass ..

after install plugin i can't use it because it should appear in navigation bar in my account ..

note : my install is success i think this problem not in my installation ..

so eny help ..

thx in advance
